### PR TITLE
feat : add margaret okore profile

### PR DESCRIPTION
**Pull request by thayeylolu**
@reshamas 

This PR adds Margaret Okore’s profile to the list of contributors.

### CODE REVIEW

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 1]
```diff
@@ -400,11 +400,12 @@ Bhopal, Madhya Pradesh, India
...
 
-<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
+<img alt="Image for contributor template" src="../../_static/contributors_2022_07/Margaret_Okore.JPG" class="rounded-circle">
 
-Margaret Chege
+Margaret Okore
 
-Kenya
+Nairobi, Kenya
+[{fab}`linkedin-in`](https://www.linkedin.com/in/margaretokore/)
```
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:408]:
```suggestion
Nairobi, Kenya

[{fab}`linkedin-in`](https://www.linkedin.com/in/margaretokore/)
```

Comment (reshamas) [reply 2022-07_sprint/sprint_parties/contributors.md:408]: I believe there needs to be a blank line.

**Comment by thayeylolu** [comment]
@reshamas 

I just included Margaret Chege's name back to the contributors. 
Initially, She was added as a contributor. I thought it was a typo because I couldn't find her picture. I found Okore's instead. 
For this reason, I deleted Chege and included Okore. What I am going to do know is - 
1. Remove Okore's details (linkedin) but keep her picture and her name
2. Include Chege's name but a blank profile

Let me know if this is okay with you.

File: 2022-07_sprint/sprint_parties/contributors.md (hunk 1 above)
Comment (thayeylolu) [reply 2022-07_sprint/sprint_parties/contributors.md]: I concluded the blank line

**Comment by reshamas** [comment]
> @reshamas
> 
> I just included Margaret Chege's name back to the contributors. Initially, She was added as a contributor. I thought it was a typo because I couldn't find her picture. I found Okore's instead. For this reason, I deleted Chege and included Okore. What I am going to do now is -
> 
> 1. Remove Okore's details (linkedin) but keep her picture and her name
> 2. Include Chege's name but a blank profile
> 
> Let me know if this is okay with you.


@thayeylolu Thanks for the PR.  Can we:
1) Move Maggie Chege up to the "Returning Contributors" (she participated in past scikit-learn sprints)
2) For both Maggie's, can you remove the "I am a new contributor to PyMC" text?  We don't have it for the other folks.

Thanks.

**Comment by reshamas** [comment]
Thanks @thayeylolu .  This PR looks good!

**Pull request merged**